UserInterface.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// User interraction of the application.
+    /// </summary>
+    static public class UserInterface
+    {
+        private const string notNumericMessage = "Please input numeric value.";
+        private const string pressAnyKeyMessage = "Press any key to continiue.";
+        private const string notInRangeMessage = "The value is not within range.";
+
+        /// <summary>
+        /// Gets numeric input from the user.
+        /// </summary>
+        /// <param name="prompt">Text to display for the user.</param>
+        /// <returns>Returns user input as int value.</returns>
+        public static int GetNumericInput(string prompt)
+        {
+            while(true)
+            {
+                Console.Clear();
+                Console.WriteLine(prompt);
+                string? value = Console.ReadLine();
+                bool success = int.TryParse(value, out int result);
+                if(success)
```

Comment:
you can put the tryParse in the if, there is no need to create a variable for the tryParse result.

Reply:
Put tryParse in if statement and removed unnecessary bool value.